src/app/app-modules/core/components/app-header/app-header.component.ts

```
@@ -100,6 +107,23 @@
     // }
     if (this.isAuthenticated) {
       this.fetchLanguageSet();
+      this.auth.getUserDetails().subscribe((res: any) => {
+        // console.log(">>>>>>", res);
+        if (res.statusCode == '200') {
+          if (res.data.previlegeObj && res.data.previlegeObj[0]) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)